backend/src/test/java/ai/verta/modeldb/HydratedServiceTest.java

```diff
@@ -126,8 +125,7 @@ public static void setServerAndService() throws Exception {
       channelBuilder.intercept(authClientInterceptor.getClient1AuthInterceptor());
       client2ChannelBuilder.intercept(authClientInterceptor.getClient2AuthInterceptor());
     }
-    deleteEntitiesCron =
-        new DeleteEntitiesCron(authService, roleService, CronJobUtils.deleteEntitiesFrequency);
+    deleteEntitiesCron = new DeleteEntitiesCron(authService, roleService, 100);
```

Comment:
wrong value was used here, because this parameter is not frequency, but updateLimit. Also it looks like this public constant is used only in tests, probably better to use some local value